main.go

```diff
@@ -36,15 +36,17 @@ var (
 		"stable": {
 			ManifestVersion: "0.1",
 			Istio: []string{
-				"https://storage.googleapis.com/knative-releases/serving/previous/v0.3.0/istio.yaml",
-				"https://storage.googleapis.com/projectriff/istio/istio-riff-knative-serving-v0-3-0-patch.yaml",
+				"https://storage.googleapis.com/knative-releases/serving/previous/v0.4.0/istio.yaml",
```

Comment:
I published `
https://storage.googleapis.com/projectriff/istio/istio-riff-knative-serving-v0-4-0-patch.yaml` which is exactly the same as the 0.3.0 patch.